app/session.server.ts

```diff
@@ -44,11 +88,27 @@ export async function getCurrentUser(session: Session) {
     return null;
   }
 
+  let decodedClaims: DecodedIdToken | undefined;
+
   try {
-    const decodedClaims = await auth.verifyIdToken(idToken);
-    return decodedClaims;
+    decodedClaims = await auth.verifyIdToken(idToken);
   } catch (err) {
     // The session isn't valid anymore so return undefined
     return null;
   }
+
+  const githubUserData = await getGitHubUserDataFromClaims(
```

Comment:
One of the tricky things about the Firebase + GitHub auth is that GitHub doesn't actually tell us the user's `login`, which is what we use in the URL path to their profile. Because of that, we must make an API call to the GitHub API to access it.

In a future PR, I will actually store this field on a "User" entity, so that we don't have to call the GH API on every call to `getCurrentUser`.